package.json

```diff
@@ -59,6 +59,19 @@
       "project": "./tsconfig.json"
     },
     "rules": {
+      "@typescript-eslint/comma-dangle": [
+        "error",
+        {
+          "arrays": "always-multiline",
+          "objects": "always-multiline",
+          "imports": "always-multiline",
+          "exports": "always-multiline",
+          "functions": "only-multiline",
+          "enums": "always-multiline",
+          "generics": "always-multiline",
+          "tuples": "always-multiline"
+        }
+      ],
```

Comment:
> (What I really want, but which ESLint does not support, is a mode which
requires dangling commas on functions that have two or more arguments,
and allows but does not require them on functions that take only a
single argument.)

💯